package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "root",
+  "private": true,
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
+    "lint": "lerna run lint",
+    "lint:fix": "lerna run lint:fix",
+    "test": "lerna run test --parallel"
```

Comment:
Where the magic happens.

Reply:
I think the lint commands can be done in parallel